quicklog/examples/macros.rs

```diff
@@ -46,5 +47,5 @@ fn main() {
         s_0, s_1, s_2, s_3, s_4, s_5, s_6, s_7, s_8, s_9, s_10
     );
 
-    flush!();
```

Comment:
previously this just printed "hello world! 2 3 4" and did not print the other lines